.github/workflows/upstream-release.yaml

```diff
@@ -0,0 +1,42 @@
+name: Snap upstream releases
+
+on:
+  workflow_dispatch:  # Allow manual run
+  schedule:
+    - cron: "0 6 * * *"  # Every day at 06:00 UTC
+
+concurrency: ${{ github.workflow }}
+
+permissions:
+  contents: write
+  packages: write
+  issues: write
+
+jobs:
+  sync-gh-releases:
+    uses: lengau/snappify/.github/workflows/sync-gh-releases.yaml@v0.0.2
```

Comment:
this... seems hacky 😅

Reply:
I originally incorporated the workflow into this project, but I realised it would be a pretty good generic workflow for snap releases if upstream does releases using GitHub's release mechanism.

Reply:
(But yes, very hacky. I'd love to figure out a better mechanism to make that generic later, but trying to include it as an action was a huge mess.)